src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -146,16 +154,34 @@ public void formats_inAlphanumeric_success() throws ParseException {
         String formats = "java js css 7z";
         String input = DEFAULT_MANDATORY_ARGS + String.format("-formats %s", formats);
         CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
-
+        Assert.assertTrue(cliArguments instanceof ConfigCliArguments);
         List<String> expectedFormats = Arrays.asList("java", "js", "css", "7z");
         Assert.assertEquals(expectedFormats, cliArguments.getFormats());
     }
 
+    @Test
+    public void parse_validRepoLocations_repoConfigurationListCorrectSize() throws ParseException, IOException {
+        String input = String.format("-repos %s %s", TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
```

Comment:
Can I have one more test - 
What if user type the input with quotes?

E.g. -repos "C:\Users\Repo Sense\Desktop\GitHub\RepoSense" "https://github.com/reposense/RepoSense/"? 
Would it still work?